authentication/migrations/0001_initial.py

```diff
@@ -19,8 +19,10 @@ class Migration(migrations.Migration):
         migrations.CreateModel(
             name='Profile',
             fields=[
-                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
-                ('user', models.OneToOneField(on_delete=django.db.models.deletion.CASCADE, related_name='profile', to=settings.AUTH_USER_MODEL)),
+                ('id', models.AutoField(auto_created=True,
```

Comment:
Please don't format all the migrations every time. That blows up the PR size and makes it harder to review

Reply:
I know; I have an autopep8 precommit hook, so as long as everyone doesn't change them it should be fine.